assignment-3/src/CHeader.js

```diff
@@ -0,0 +1,23 @@
+//import React from "react";
+import "./styles.css";
+import image from "./assets/images/logo.jpg";
+
+const CHeader = ()=>{
+    return (
+        <div className='navbar'>
+        <div className='image__container'>
+          <img src={image} />
```

Comment:
img tag must have alt text

Reply:
added alt text,parcel cache is also added still showing cache changes in vscode ..not sure about the issue